CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ file(GLOB LIB_SRC
 
 file(GLOB ALL_TEST
         "./src/test/test.cpp"
+        "./src/test/test_btf.cpp"
```

Comment:
Not sure I follow. Currently the list is sorted so that "." comes before "_". To be consistent, I either need to move test.cpp to the end of the list, or leave it as is.